package.json

```diff
@@ -1,32 +1,13 @@
 {
   "name": "esphome-webserver",
-  "version": "2.0.5",
-  "main": "index.js",
```

Comment:
I don't believe this is used unless this repo/package is being published to the NPM registry.